src/distance_finder.cpp

```diff
@@ -333,11 +371,9 @@ class DistanceFinder {
         lidarCalculations lidarCalc;
         double radius = lidarCalc.calculateRadius(circle_points, min_circle_pts_p);
         ROS_DEBUG_STREAM(TAG << "r = " << radius << ", close pt = " << closest_distance);
-        // TODO: change below radius validation so that we only compare against the correct prop type
-        if (!((radius > marker_radius_p - prop_range_p  && radius < marker_radius_p + prop_range_p ) || (radius > buoy_radius_p - prop_range_p && radius < buoy_radius_p + prop_range_p))) {
-            // if the prop doesn't fit a radius range, then it doesn't have a valid radius
-            // to be more specific, get label for prop then specify what radius to compare to based on that
-            ROS_DEBUG_STREAM(TAG << "Calculated radius: " << radius << " does not match the expected radius");
+
+        if (!validateRadius(prop_angles_msg_.prop_label, radius)) {
+            ROS_WARN_STREAM(TAG << "Calculated radius does not match expected " << prop_angles_msg_.prop_label << " radius");
```

Comment:
Can we move this print statement to the validateRadius function so we can print the expected and calculated radius values in the statement?